test/models/user_test.rb

```diff
@@ -2,24 +2,81 @@
 
 require 'test_helper'
 
-# Test users.
+# Test user model.
 class UserTest < ActiveSupport::TestCase
-  # Test validations.
+  # User tests setup
+  def setup
+    @user = User.new(name: 'test User_name',
+                     email: 'test@email.com',
+                     password: 'foobar',
+                     password_confirmation: 'foobar')
+  end
+
+  # Test new user
 
-  test 'should succeed empty user name' do
-    user = User.new(name: '', email: 'test@email.com')
-    refute user.valid?, 'user is valid without a name'
-    assert_not_nil user.errors[:name], 'no validation error for name present'
+  test 'should succeed name sould be pressent' do
```

Comment:
`should fail validation with missing name` Apply this convention everywhere. Just see the model tests in IO and DM

Reply:
also try to avoid typos, e.g. `sould`